hedera-node/hedera-app/src/main/java/com/hedera/node/app/workflows/handle/HandleContextImpl.java

```diff
@@ -444,17 +445,21 @@ public <T> T dispatchPrecedingTransaction(
         if (category != TransactionCategory.USER) {
             throw new IllegalArgumentException("Only user-transactions can dispatch preceding transactions");
         }
-        if (stack.depth() > 1) {
-            throw new IllegalStateException(
-                    "Cannot dispatch a preceding transaction when a savepoint has been created");
-        }
-
-        if (current().isModified()) {
-            throw new IllegalStateException("Cannot dispatch a preceding transaction when the state has been modified");
-        }
+        // In mono-service preceding records is just the way we arrange the records,
```

Comment:
Because we charge fees before this and the ACCOUNTS state has modifications. 🙂 . So we fail here when we see `current().isModified()`

Reply:
And if I move the `finalizeHollowAccounts` all the way up. Then because we dispatch hollow accounts finalizations before autoCreations, it fails while dispatching preceding auto-creation transaction. 😞